### ISSUE TITLE
Inconsistency in whitespace handling between macro and swc plugin for 5.0.0-next.0

### ISSUE THREAD

**Issue by AndrewIngram** [lingui/swc-plugin#98]
We've discovered that a multi-line string with lines ending with nbsp behave differently with the macro/extraction and the swc plugin.

Example:

```jsx
<Trans>
  Line ending in non-breaking space.&nbsp;
  <strong>text in element</strong>
</Trans>
```

Extracts (correctly, from what I can tell) as
```
Line ending in non-breaking space. <0>text in element</0>
```
with a message ID of `CJuEhi`

But at runtime (via the swc plugin) it's looking for a string of ID `9gH3FD`, which is the same ID you get during extraction if you remove the nbsp from the source string, i.e:

```jsx
<Trans>
  Line ending in non-breaking space.
  <strong>text in element</strong>
</Trans>
```

Which extracts as:
```
Line ending in non-breaking space.<0>text in element</0>
```

**Comment by thekip** [comment]
What version of SWC plugin do you use?

**Comment by AndrewIngram** [comment]
This is with `5.0.0-next.0`

**Comment by thekip** [comment]
it seems some SWC internals give a different result. I will take a look

**Comment by thekip** [comment]
fixed

**Comment by andrii-bodnar** [comment]
Available in [5.0.0-next.1](https://github.com/lingui/swc-plugin/releases/tag/5.0.0-next.1).
